sass/abstracts/_colors.scss

```diff
@@ -6,6 +6,7 @@ $blue: #003b63;
 $red: #bd2b2e;
 $grey: #303030;
 $gray: $grey;
+$grey--alt: #353530;
```

Comment:
We shouldn't have an alt, sometimes the sketch file handles color wrong - @melchoyce Should this be `#303030;`, or is this actually another color?

Reply:
Oops yeah, this should be #303030. Every color should match the existing variables in the theme.

Reply:
Meant to point that out in the PR and get your feedback. Glad you caught it. Went ahead and switched it.